charts/tails-server/values.yaml

```diff
@@ -51,7 +51,7 @@ extraEnvFrom: []
 #   storageClass: netapp-file-standard
 #   mountPath: /data
 persistence:
-  enabled: false
+  enabled: true
   # If set, use an existing PVC and ignore size/accessModes/storageClass
   existingClaim: ""
   size: 10Gi
```

Comment:
Might want to make this default smaller: tails storage grows quickly, however having a big PVC from the beginning might not be good for someone who is just testing things out.